docs/current/sailbot_workspace/workflow.md

```diff
@@ -99,58 +99,95 @@ launch configuration.
 
-1. Run the package(s) that your changes are in: the run commands for each package should be documented in their READMEs,
-   but in general they can be run using a VS Code or CLI command:
-    - VS Code:
-        - Launch files: `ROS: Run a ROS launch file (roslaunch)`
-        - Nodes: `ROS: Run a ROS executable (rosrun)`
-    - CLI:
-        - Launch files: `ros2 launch <package> <launch file>`
-        - Nodes: `ros2 run <package> <executable>`
-
-        ??? tip "CLI features"
-
-            There are many commands that can be autocompleted in the terminal.
-            Take advantage of this so that you run commands faster and memorize less syntax.
-            If there is only one possibility, pressing tab once will complete it.
-            If there is more than one possibility, pressing tab again will list them out.
-
-            Some tab completion use cases:
-
-            - View available commands: lists all `ros2` commands
-
-                ```console
-                $ ros2 <tab><tab>
-                action                          extension_points                multicast                       security
-                bag                             extensions                      node                            service
-                ...
-                ```
-
-            - Complete commands: runs `ros2 launch local_pathfinding main_launch.py`
-
-                ```console
-                $ ros2<tab>la<tab>loc<tab>m<tab>
-                ```
-
-            - Navigate to directories: runs `cd .devcontainer/config` from the root directory of Sailbot Workspace
-
-                ```console
-                $ cd .d<tab>c<tab>
-                ```
-
-            Furthermore, navigate past commands with ++arrow-up++ and ++arrow-down++ and search through them with ++ctrl+r++.
-
-2. Debug your changes if they aren't behaving how you expect by setting breakpoints and running one of our launch
-   configurations; launch configuration types:
-    - Launch: runs the desired launch file or executable
-        - For launch files, `ROS: Launch`
-        - For C++ executables, `C++ (GDB): Launch`
-    - Attach: attaches to a running executable
-        - `ROS: Attach`
-3. Run lint and test tasks to make sure you changes will pass our CI:
-    - `ament lint`
-    - For C++ packages, `clang-tidy`
-    - `test`
+### Lint and Test
+
+Run lint and test tasks to make sure you changes will pass our CI:
+
+- `ament lint`
+- For C++ packages, `clang-tidy`
+- `test`
+
+In addition to VS Code tasks, the :fontawesome-solid-flask: **Testing** tab on the VS Code primary sidebar
+contains individual tests. One can run specific unit tests by clicking the :material-play-outline: **Run Test**
+icon beside the test name.
+
+<img
+    alt="VS Code Test Tab" src="../../../assets/images/sailbot_workspace/workflow/vscode_testing_tab.png"
+    style="display: block; margin-left: auto; margin-right: auto; width: 75%;"
+/>
```

Comment:
This is pretty big on my monitor. Also it is outdated and has some errors. Here is a better screenshot
<img width="210" alt="image" src="https://github.com/UBCSailbot/docs/assets/65756895/9486583c-0e12-4673-9b83-9d4a2b55f0a4">